src/v2/Apps/Partner/Components/PartnerHeader/PartnerHeaderAddress.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import styled from "styled-components"
+import { uniq } from "lodash"
+import { PartnerHeader_partner } from "v2/__generated__/PartnerHeader_partner.graphql"
+
+export const TextWithNoWrap = styled.span`
+  white-space: nowrap;
+`
+export const PartnerHeaderAddress: React.FC<
+  PartnerHeader_partner["locations"]
+> = ({ edges }) => {
+  return (
+    <>
+      {uniq(edges.map(edge => edge.node.city))
+        .map(city => city.trim())
```

Comment:
This logic it's a copy of the previous app, but it has a strange behavior: if we take ATHR partner for example it has 4 addresses with cities: `["Jeddah", "Jeddah", " Jeddah", "Jeddah "]` so, after `uniq` function call   I render 3 cities: `["Jeddah", " Jeddah", "Jeddah "]`. Is it okay or maybe use trim before `uniq` function call?

Previous version:
https://github.com/artsy/force/blob/953d162ee90f7a2223d34ddf902a1c888f0bd2f8/src/desktop/collections/partner_locations.coffee#L27-L30

Reply:
Agree-- let's trim first!

Reply:
@sweir27 Lets fix that in a separate PR. I can't apply this changes from home :) Also it would be good to deploy this branch by tomorrow if everything is okay.